catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/FormulaEditorTestUtil.java

```diff
@@ -135,6 +136,18 @@ public static void testDoubleParameterFunction(Functions function, InternTokenTy
 		assertEquals(expected, parseTree.interpretRecursive(testSprite));
 	}
 
+	public static void testDoubleParameterFunction(Functions function, InternTokenType firstInternTokenType,
+			String firstParameter, InternTokenType secondInternTokenType, String secondParameter, Double upperBound,
+			Double lowerBound, Sprite testSprite) {
+
+		List<InternToken> internTokenList = buildDoubleParameterFunction(function, firstInternTokenType,
+				firstParameter, secondInternTokenType, secondParameter);
+		FormulaElement parseTree = new InternFormulaParser(internTokenList).parseFormula();
+
+		assertNotNull(parseTree);
+		assertTrue(((double) parseTree.interpretRecursive(testSprite) <= upperBound) && ((double) parseTree.interpretRecursive(testSprite) >= lowerBound));
```

Comment:
you can use the hamcrest [isCloseTo](http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/number/IsCloseTo.html) matcher here to assert that the number is somewhere close within a certain bound.

summoning @84n4n4 on this one though.